src/tests/controllers/index.js

```diff
@@ -1,2 +1,3 @@
 import './CommentController.test';
 import './AuthPassportController.test';
+import './Articles.test';
```

Comment:
I'd suggest to not mix the naming convention, if you're testing a controller, it is better to name the file that tests it the same as its name

Reply:
Thanks, Resolved